mastodon_bot_combined/src/bot2.js

```diff
@@ -0,0 +1,63 @@
+const M = require("./mastodon");
+
+function toot(content, id) {
+  const params = {
+    status: content
+  }
+  if (id) {
+    params.in_reply_to_id = id;
+  }
+  M.post('statuses', params, (error, data) => {
```

Comment:
We might as well refactor everything to use `async` functions rather than callbacks?

Reply:
I've just copy/pasted code from the individual bots to post the PR. I can indeed rework the code into Async/Await code.